### PR TITLE
Devas 2102 set theme

### PR DESCRIPTION
**Pull request by pascalhuynh**
Set project theme (2 main colors) from Assembl administration

### CODE REVIEW

File: assembl/static2/js/app/utils/translations.js [hunk 1]
```diff
@@ -1787,6 +1792,11 @@ const Translations = {
       slugWarning:
         "This modification causes a change in the address of the debate. Please take this change into account in further communications. To insure the continuity of the service, the previous address will still be operating.",
       invalidSlug: "Special characters are not allowed",
+      theme: {
+        themeColorsTitle: "Theme colors",
+        firstColor: "Primary color",
```
Comment (gary-bluenove) [inline assembl/static2/js/app/utils/translations.js]: technically I would put a 'u' in colour but it depends if we're using american english or not

Comment (pascalhuynh) [reply assembl/static2/js/app/utils/translations.js]: I learnt english in the US @gary-bluenove and I am more of a burger guy than fish and chip :)

Comment (felixsaintgenis) [reply assembl/static2/js/app/utils/translations.js]: america-fuck-yeah-memes-com-13888225

File: assembl/static2/css/variables.scss [hunk 2]
```diff
@@ -18,9 +18,9 @@ $blue: #7490ca;
 $purple: #5404d7;
 $purple-light: #e7e6f5;
 $yellow: #f0ab2b;
-$cyan: #0095b2;
-$cyan-transparent: #0094b291;
-$cyan-light: #3aaece;
+$cyan: green; // TO_DELETE and temporary replace it with 'green'
```
[the remark this answers is not in the document]
Comment (pascalhuynh) [reply assembl/static2/css/variables.scss]: cleaned

File: assembl/static2/css/components/multiColumn.scss [hunk 3]
```diff
@@ -164,40 +150,21 @@ $column-synthesis-height: 320px;
         font-family: $font-family3;
       }
     }
-  }
-}
-
-.multi-column-container {
-  display: flex;
-  justify-content: center;
-
```
[the remark this answers is not in the document]
Comment (pascalhuynh) [reply assembl/static2/css/components/multiColumn.scss]: cannot reproduce the issue (BTW @felixsaintgenis thanks for the fake screenshot jk :))

[the remark this answers is not in the document]
Comment (pascalhuynh) [reply assembl/static2/css/components/multiColumn.scss]: Fixed with BH-345

File: assembl/static2/css/components/synthesisPreview.scss [hunk 4]
```diff
@@ -12,10 +12,6 @@
     width: calc(100% - 40px);
   }
 
-  .light-title-4 {
-    color: $second-color;
```
[the remark this answers is not in the document]
Comment (pascalhuynh) [reply assembl/static2/css/components/synthesisPreview.scss]: Fixed with BH-345

**Comment by pascalhuynh** [comment]
* Committed new PR and bug fixes
* DEVAS-2207: fix color picker text field issue
* Rebased with the latest version of develop

Next step: 
* waiting for @ayyazdaniaryan fixes to be pushed
* test a deployment on sandbox
* fill first and second color attributes in the database (preferences) @gary-bluenove 
* merge on develop

**Comment by pascalhuynh** [comment]
@ayyazdaniaryan the issue preventing the style.css to be generated as a stylesheet is now fixed. Now let's setup everything so that we can deploy that branch on sandbox and UAT.

**Pull request merged**